spec/lib/importers/case_importer_spec.rb

```diff
@@ -11,6 +11,13 @@
     allow(case_importer).to receive(:email_addresses_to_users) do |_clazz, comma_separated_emails|
       create_list(:volunteer, comma_separated_emails.split(",").size, casa_org_id: casa_org_id)
     end
+
+    # next_court_date in casa_cases.csv needs to be a future date
+    travel_to Date.parse("Sept 15 2022")
+  end
+
+  after do
+    travel_back
```

Comment:
The changes here and in `imports_spec` are not related but I broke these tests in my previous PR.